.travis.yml

```diff
@@ -26,7 +26,7 @@ notifications:
     on_success: change
     on_failure: always
     on_start: never
-sudo: false
+sudo: required
```

Comment:
Wow, if that's all it took to fix this, we've spend a lot of work dealing with travis problems...